sass/routes/_signup.scss

```diff
@@ -515,7 +515,7 @@ body {
     border: 1px solid #dddddd;
     margin: 0 0 20px;
     border-radius: 4px;
-    overflow: hidden;
+    max-height: 448px;
```

Comment:
This is a problem with mobile webview. Because this will cause two scrolls on the body. Can you change this to be based on calc with percentage and subtract the header and footer spaces. If not don't worry we will have one of our engineers add a commit at the end for the css changes.

Reply:
@sudheerDev made a change to use calc() but even then, there'll be 2 scrolls on the body on some mobile screens. The list container is also a bit too small. It seems like the footer (also mentioned on JIRA) and some other items take up a lot of space on the page.